client/post-editor/media-modal/detail/detail-preview-media-file.tsx

```diff
@@ -35,10 +35,14 @@ const EditorMediaModalDetailPreviewMediaFile: React.FC< Props & LocalizeProps >
 		);
 	}
 
+	const mediaUrl = url( item, {} );
+	if ( ! mediaUrl ) {
+		return null;
+	}
 	return (
 		<MediaFile
 			component={ component }
-			src={ url( item, {} ) }
+			src={ mediaUrl }
```

Comment:
Ideally, I'd recommend doing this in another PR.

Reply:
We unfortunately have to do this here since the types of `url` are changing -- we'd either have to not fix the types in the other file, or accept a failing typescript build